.jenkins/library/vars/tests.groovy

```diff
@@ -51,13 +51,12 @@ def ACCTest(String label, String compiler, String build_type, List extra_cmake_a
                 if (fresh_install) {
                     sh  """
                         sudo bash scripts/ansible/install-ansible.sh
-                        # Run ACC Playbook
-                        for i in 1 2 3 4 5
-                        do
-                            sudo \$(which ansible-playbook) scripts/ansible/oe-contributors-acc-setup.yml && break
-                            sleep 60
-                        done
                         """
+                    retry(5) {
+                        sh """
+                            sudo ansible-playbook scripts/ansible/oe-contributors-acc-setup.yml
```

Comment:
Don't we need the sleep for 60 seconds anymore?

Reply:
I was thinking that it was not useful as there's nothing to wait on -- and any service/infra issues would likely last longer than the 4 minutes that the sleeps add. But then again, the extra 4 minutes won't hurt anyways.

Reply:
Ran tests here: https://openenclaveci.westus.cloudapp.azure.com/blue/organizations/jenkins/OpenEnclave%2FAzure-Linux/detail/Azure-Linux/194/pipeline/74